src/components/node/NodeLogs.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { invoke } from '@tauri-apps/api/tauri';
+import { LogsContainer, LogsOutput, LogsHeader } from '../../styles/NodeLogsStyles';
+import { NodeDetails } from '../../hooks/useNodeManagement';
+
+interface NodeLogsProps {
+  selectedNode: NodeDetails;
+  onClose: () => void;
+}
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ selectedNode }) => {
+  const [logs, setLogs] = useState<string>('');
```

Comment:
so in some cases you use <string | null>(null) and here its (''). Any reason for this ? 
Think we can use just <string> in other cases also as "" will return false same as null when tested

Reply:
updated, switched to use only strings